### ISSUE TITLE
Update to 7.6 prevents access to posts and pages in MS Edge v44

### ISSUE THREAD

**Issue by murray333** [WordPress/gutenberg#20635]
Since updating to 7.6, I cannot edit or create posts and pages.  Clicking on "edit" produces nothing until I get the "This page is having a problem loading" report.

I am using Microsoft Edge browser (Version 44.18362.449.0) in Windows 10.

Deactivating Gutenberg allows me to access posts and pages, but means I cannot edit with the block editor.

I have now checked and this problem does not occur when using Firefox.

Please advise.  Many thanks.

**Comment by talldan** [comment]
Hi @murray333, there were some recent issues that caused the editor not to load, but those were fixed in 7.5 (https://github.com/WordPress/gutenberg/pull/20110).

> I am using Microsoft Edge browser (Version 44.18362.449.0) in Windows 10.

I'd recommend updating Edge, the latest version should be v80 (the version numbering system changed recently, v79 was the start of the new numbering system so it looks like you're a couple of versions behind). 

I gave it a test using Edge 80 and things seemed to work fine with 7.6. Let us know how you get on.

**Comment by talldan** [comment]
Oh, I'm not sure what the story is with Edge rollout, but I wasn't aware that this might be a staged rollout, so apologies if you're unable to upgrade @murray333.

**Comment by murray333** [comment]
Hi Daniel

Many thanks for the prompt replies.  I suspected this may have been a compatibility with Microsoft Edge problem and this is confirmed, as I have now tested that everything is OK using Firefox or Chrome.

My understanding is that Edge can only be updated through Windows Update and this had been updated only yesterday.

I note that my PC received notice of a new Cumulative Update overnight and I will see if this effects Edge.

Otherwise I'll avoid using Edge until I can test Gutenberg with the latest version, whenever that is rolled out to me.

Thank you, Martin

 

  _____  

Martin Minter -  <mailto:martin@minter-roff.co.uk> martin@minter-roff.co.uk

  _____

**Comment by murray333** [comment]
This problem seems to have been solved, subject to extensive checking.
I have now upgraded to Edge 80, but using Windows Update did not achieve this.  It seems Windows 10 was updating, but leaving Edge in the Dark Ages!
I successfully updated Edge by using the following link (it took some searching!): -
[https://blogs.windows.com/msedgedev/2020/01/15/upgrading-new-microsoft-edge-79-chromium/]
Thank you for your help and I hope this may also be of use to others.
